syntax = "proto3";

package dragongem.trace;

message BBInterval {
  map<uint64, uint64> freq = 1;
  uint64 inst_start = 2;
  uint64 inst_end = 3; // Inclusive.
}
